docs/api/users/update.md

```diff
@@ -1,7 +1,133 @@
 sidebar_label: Update
+keywords:
+  - OpenHIM
+  - API
+  - User
+  - Update
+description: Update an OpenHIM User via the API
 ---
 
-Coming soon ...
+## Update user
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+To update an existing user record, you will need to make a TLS request to the OpenHIM API for the below method and endpoint and supply the updated JSON object for the user record.
+
+```curl
+Method: PUT
+Endpoint: {openhim_url}:8080/users/:userEmail
+Payload: JSON object of the user record
+```
+
+## Example
+
+<Tabs
+  defaultValue="lang"
+  values={[
+    { label: 'Languages', value: 'lang' },
+    { label: 'Sample Data', value: 'sample' }
+  ]
+}>
+<TabItem value="lang">
+
+  Before we can send our request to the OpenHIM API we need to ensure that we construct our valid HTTP headers to successfully authenticate with the OpenHIM API. 
+
+  <Tabs
+    defaultValue="nodejs"
+    values={[
+      { label: 'NodeJS', value: 'nodejs' },
+      { label: 'Bash', value: 'bash' }
+    ]
+  }>
+  <TabItem value="nodejs">
+
+  Copy the below code at the bottom of your nodejs script that handles the authentication of the OpenHIM headers as described in the [authentication section](../introduction/authentication.md).
+
+  Replace the `openhimOptions` values with the correct implementation details and supply the `SampleData` payload to submit
+
+  ```javascript
+  // append below code to the "openhim-api.js" script containing the authentication methods. 
+  // This is described within the authentication section
+
+  (async () => {
+    const openhimOptions = {
+      apiURL: 'https://localhost:8080',
+      apiEndpoint: '/users/userEmail',
+      username: 'root@openhim.org',
+      password: 'openhim-password',
+      rejectUnauthorized: false
+    }
+    const SampleData = 'SampleData'
+
+    const headers = await genAuthHeaders(openhimOptions)
+
+    const options = { 
+      method: 'PUT',
+      url: `${openhimOptions.apiURL}${openhimOptions.apiEndpoint}`,
+      rejectUnauthorized: openhimOptions.rejectUnauthorized,
+      headers: headers,
+      body: SampleData,
+      json: true
+    }
+
+    request(options, (error, response, body) => {
+      if (error) throw new Error(error)
+
+      console.log({
+        statusCode: response.statusCode,
+        body
+      })
+    })
+  })()
+  ```
+
+  Execute the below command in your terminal to run the nodejs script
+
+  ```bash
+  node openhim-api.js
+  ```
+
+  </TabItem>
+    <TabItem value="bash">
+
+  Ensure that you have created your bash script to construct the HTTP authentication headers and send the request to the OpenHIM API as described in the [authentication section](../introduction/authentication.md).
+
+  Execute the below command in your terminal where the file is located with the required arguments. Replace the placeholder arguments with the correct implementation details and ensure your `SampleData.json` file exists with the updated user object.
+
```

Comment:
We are missing the bash commands here for generating the hashed password. Its useful to supply it here as well so its easy to copy/past the example and have it working out the box